README.md

```diff
@@ -61,8 +61,10 @@ postcss: (ctx) => {
 
 | Name | Description | Default |
 | ---- | ----------- | ------- |
-| **root** | Root path used to resolve layouts and includes | |
+| **root** | Root path used to resolve layouts and includes | Defaults to `webpack.options.context` which is the project root folder.|
```

Comment:
Maybe defaults to `webpack.options.context` if the `webpack` option is provided -- otherwise it should be empty I think